acceptance/tests/vault/helpers.go

```diff
@@ -160,6 +178,22 @@ func configureKubernetesAuthRoles(t *testing.T, vaultClient *vapi.Client, consul
 	}
 	_, err = vaultClient.Logical().Write(fmt.Sprintf("auth/%s/role/consul-ca", authPath), params)
 	require.NoError(t, err)
+
+	logger.Log(t, "Creating kubernetes auth role for the server-acl-init job")
```

Comment:
we now need to setup the role for server-acl-init here (instead of `configureReplicationTokenVaultSecret`) because it's needs both replication and bootstrap tokens policies